newIDE/app/src/Utils/GDevelopServices/Usage.js

```diff
@@ -276,11 +278,29 @@ export const canSeamlesslyChangeSubscription = (subscription: Subscription) => {
   return !!subscription.stripeSubscriptionId;
 };
 
-export const hasValidSubscriptionPlan = (subscription: ?Subscription) =>
-  !!subscription &&
-  !!subscription.planId &&
-  (!subscription.redemptionCodeValidUntil || // No redemption code
-    subscription.redemptionCodeValidUntil > Date.now()); // Redemption code is still valid
+export const hasMobileAppStoreSubscriptionPlan = (
+  subscription: ?Subscription
+): boolean => {
+  return (
+    // The user has a subscription registered in the backend (classic "Registered" user).
```

Comment:
I'm not sure I see the value of this comment + you have it duplicated below too.
Are you trying to say that the user is registered and their subscription is linked to their user profile? (instead of only an anonymous account with a store subscription?)